app/controllers/api/mixins/service_templates.rb

```diff
@@ -17,6 +19,14 @@ def order_service_template(id, data, scheduled_time = nil)
       def service_template_ident(st)
         "Service Template id:#{st.id} name:'#{st.name}'"
       end
+
+      def api_request_allowed?
+        standalone_api_request? ? Settings.product.allow_api_service_ordering : true
+      end
+
+      def standalone_api_request?
```

Comment:
Can we name this better? I don't understand what it means for an API request to be "standalone", maybe reverse the logic and call it `request_from_ui?`

Reply:
That's fair, I was struggling to come up with a good name, but reversing the logic and naming it `request_from_ui?` sounds better to me